.azurepipelines/BaseTools-Build-For-Publication.yml

```diff
@@ -112,7 +112,51 @@ jobs:
         tool_chain_tag: 'VS2022'
 
     - publish: $(windows_build_output_path)
-      artifact: $(windows_artifact_name)
+      artifact: $(windows_artifact_name)_x86
+      displayName: Publish To Pipeline
+      condition: SucceededOrFailed()
+
+  - job: windows_build-ARM
```

Comment:
Given the only difference in the `windows_build-x86` and `windows_build-ARM` jobs is:

1. `-a ARM` is added to `extra_parameters` to Steps/BuildBaseTools.yml
2. `$(windows_artifact_name)_x86` vs `$(windows_artifact_name)_ARM` as parameter value

That could be cleanly defined in a simple matrix and reduce a lot of this new code.

Can you look into that?

Reply:
Good call, will update that once verified functionality.

Reply:
This is updated. I also found there needs to be extra handling given now that we require valid license. Please let me know if any feedback there. Thanks.

Reply:
The commit has more info about options - [Nuget Publishing deprecation: switch from LicenseUrl to License](https://github.com/tianocore/edk2-pytool-extensions/commit/27e3877c2d53c538e396ef4438da40b74c6c16eb).

I think you can use the `--CustomLicensePath` parameter and specify the absolute path to the [LICENSE.md](https://dev.azure.com/projectmu/_git/acpica?path=/LICENSE.md) at the root of the acpica repo. As that meets the binary distribution conditions specified in the file.

Reply:
yeah, that is what i did. thanks! :)

Reply:
Well, since the publishing step is a separate job than the building step, i fetched the license.txt from BASECORE instead and specify it using `--CustomLicensePath`. But let me know if this is unorthodox.

Reply:
great, thanks!